### ISSUE TITLE
Identification error with # in url

### ISSUE THREAD

**Issue by shutter-cp** [jina-ai/reader#92]
https://vant-ui.github.io/vant-weapp/#/tabbar

**Comment by nomagick** [comment]
Hi.
I've patched Reader to allow HTTP POST and passing `url` in body.
This should fix your issue.
https://github.com/jina-ai/reader?tab=readme-ov-file#spas-with-hash-based-routing

```bash
curl -X POST 'https://r.jina.ai/' \
	-d 'url=https://vant-ui.github.io/vant-weapp/#/tabbar'
```

**Comment by nomagick** [comment]
Fixed in 873994397a257ccb77b3008e99363b40941c7240

**Comment by shutter-cp** [comment]
> Fixed in [8739943](https://github.com/jina-ai/reader/commit/873994397a257ccb77b3008e99363b40941c7240) 已修复 8739943

Either request
https://vant-ui.github.io/vant-weapp/#/tabbar
 https://vant-ui.github.io/vant-weapp/#/theme
https://vant-ui.github.io/vant-weapp/#/button

They all return the same thing

**Comment by shutter-cp** [comment]
> > Fixed in [8739943](https://github.com/jina-ai/reader/commit/873994397a257ccb77b3008e99363b40941c7240) 已修复 [8739943](https://github.com/jina-ai/reader/commit/873994397a257ccb77b3008e99363b40941c7240)
> 
> Either request 任一请求 https://vant-ui.github.io/vant-weapp/#/tabbar https://vant-ui.github.io/vant-weapp/#/theme https://vant-ui.github.io/vant-weapp/#/button
> 
> They all return the same thing他们都返回相同的东西

Solved the cache problem    x-no-cache: true

**Comment by nomagick** [comment]
> Either request
https://vant-ui.github.io/vant-weapp/#/tabbar
https://vant-ui.github.io/vant-weapp/#/theme
https://vant-ui.github.io/vant-weapp/#/button
> They all return the same thing

Thanks for reporting again, this is another bug. `x-no-cache: true` is not required now.
